docs/Wdio_Utility.md

```diff
@@ -62,76 +61,54 @@ There are a few things to note about the webdriver.io configuration provided by
 
 Then, to assist with testing, the TerraService provides the Terra global helper to make testing easier:
 
-- `Terra.viewports(name)` takes the viewport key name(s) and returns an array of { height, width } objects representing the respective terra viewport size(s).
+- `Terra.describeViewports` mocha `describe` block intended to replace the spec file's top level `describe` block. This method will run all tests within it's block at the given viewport(s). `Terra.describeViewports` blocks should not be nested and if tests need to run against different viewports then they should have their own top level `Terra.describeViewports` block. Only Terra defined viewports are accepted. This method accepts these arguments (in this order):
+    - String: The describe block name. Will automatically have viewport information included.
+    - String array: The viewport(s) to run the tests at. If the formFactor is defined that is the only viewport size that tests may be ran against.
+    - Function: The method containing the tests to run against each viewport.
+    - See [describeViewports-spec.js](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/describeViewports-spec.js) for example usage.
+- `Terra.viewports(name)` **Note - It is preferred to use Terra.describeViewports().** takes the viewport key name(s) and returns an array of { height, width } objects representing the respective terra viewport size(s).
     - Use this function to resize the browser or pass the viewport sizes to the accessibility and visual regression commands.
     - By default returns all viewports if not name key are provided.
-- `Terra.should.beAccessible()` mocha-chai convenience method that runs an axe test for the page. Takes the same arguments as the `axe()` utility.
+- `Terra.it.validatesElement()` mocha-chai convenience method that takes a screenshot and verifies the images are within the specified mis-match tolerance and performs accessibility validation. Note: this method provides its own mocha it test case. Also, since viewports isn't accepted in this method, you need to [test in multiple viewports](https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/describeViewports-spec.js). This method accepts these arguments (in this order):
+    - String (optional): the test case name. Default name is 'default'
+    - Object (optional): the test options. Options include selector, misMatchTolerance and axeRules:
+         - selector: the element selector to take a screenshot of. Defaults to the global terra.selector.
+         - misMatchTolerance: number between 0 and 100 that defines the degree of mismatch to consider two images as identical, increasing this value will decrease test coverage. Defaults to the global visualRegression.compare.misMatchTolerance.
```

Comment:
Defaults to the global `visualRegression.compare.misMatchTolerance`.

Reply:
Done in 04987d77e2d549eee0d8c03ad3f07fe0fa4103c2